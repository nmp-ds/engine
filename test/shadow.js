import { expect, test } from 'vitest';
import { setup } from './_helpers.js';

setup();

test('shadows work', async ({ uno }) => {
  const classes = ['shadow-s', 'shadow-m', 'shadow-l', 'shadow-xl'];
  const { css } = await uno.generate(classes);
  expect(css).toMatchSnapshot();
});

test('drop shadows', async ({ uno }) => {
  const classes = ['drop-shadow-s', 'drop-shadow-m', 'drop-shadow-l', 'drop-shadow-xl'];
  const { css } = await uno.generate(classes);
  expect(css).toMatchSnapshot();
});
