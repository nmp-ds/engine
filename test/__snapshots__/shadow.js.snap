// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`shadows work 1`] = `
"/* layer: default */
.shadow-large{box-shadow:var(--w-shadow-large, 0 6px 8px rgba(0, 0, 0, .2), 0 10px 20px rgba(0, 0, 0, .2));}
.shadow-medium{box-shadow:var(--w-shadow-medium, 0 3px 8px rgba(0, 0, 0, .2), 0 3px 6px rgba(0, 0, 0, .2));}
.shadow-small{box-shadow:var(--w-shadow-small, 0 1px 6px rgba(0, 0, 0, .2), 0 1px 1px rgba(0, 0, 0, .2));}
.shadow-xlarge{box-shadow:var(--w-shadow-xlarge, 0 9px 12px rgba(0, 0, 0, .2), 0 14px 28px rgba(0, 0, 0, .2));}"
`;
